Sources/Sentry/SentryBreadcrumbTracker.m

```diff
@@ -134,12 +134,35 @@ - (void)swizzleSendAction
     SentrySwizzleInstanceMethod(UIApplication.class, selector, SentrySWReturnType(BOOL),
         SentrySWArguments(SEL action, id target, id sender, UIEvent * event), SentrySWReplacement({
             if (nil != [SentrySDK.currentHub getClient]) {
-                NSDictionary *data = nil;
+                NSMutableDictionary *data = nil;
                 for (UITouch *touch in event.allTouches) {
                     if (touch.phase == UITouchPhaseCancelled || touch.phase == UITouchPhaseEnded) {
-                        data = @ { @"view" : [NSString stringWithFormat:@"%@", touch.view] };
+                        data = @ { @"view" : [NSString stringWithFormat:@"%@", touch.view] }
+                                   .mutableCopy;
+
+                        if ([sender isKindOfClass:UIView.class]) {
+                            UIView *senderView = sender;
+                            if (senderView.tag > 0) {
+                                [data setValue:[NSNumber numberWithInteger:senderView.tag]
+                                        forKey:@"tag"];
+                            }
```

Comment:
`m`: I wonder if adding the tag actually makes sense. Would it help you when you debug your app?

Reply:
I can thing on some very specifics cases, like dynamic generated fixed UI.